pages/about.html

```diff
@@ -5,8 +5,20 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
+  <link href="../styles/style.css" rel="stylesheet">
 </head>
 <body>
-  
+  <section class="flexbox-container">
+    <div class="info"><h1>My Interest</h1></div>
+    <div class="box">
+      <a href="https://static.vecteezy.com/system/resources/previews/003/536/355/original/mountain-snowboarding-activity-free-vector.jpg"> <p>Snowboarding</p></a>
+    </div>
+    <div class="box">
+      <a href="https://thumbs.dreamstime.com/z/hikers-man-woman-climb-hill-cartoon-vector-illustration-isolated-couple-white-background-characters-hiking-traveling-196872121.jpg"> <p>Hiking</p></a>
+    </div>
```

Comment:
To make it easier to read your HTML, consider separating each element on its own line with indentation, like:
```html
<div class="info">
    <h1>My Interest</h1>
</div>
```